### PR TITLE
dont send flags when parameter is empty

### PR DESCRIPTION
**Pull request by kontrollanten**
solves #9

### CODE REVIEW

File: orb.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ commands:
                     -n << parameters.upload_name >> \
                     -t << parameters.token >> \
                     -y << parameters.conf >> \
-                    -F << parameters.flags >> \
+                    -<<# parameters.flags >> -F << parameters.flags >> <</ parameters.flags >> \
```
Comment (thomasrockhu) [inline orb.yml]: I might be looking at the wrong documentation, but I believe this needs `parameters.flags` to evaluate to a boolean-like thing: https://circleci.com/docs/2.0/reusing-config/#boolean

Unless, of course, empty string evaluates to a falsy value, I think it needs to be in the set of ```y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF```
accordingn to https://yaml.org/type/bool.html

Comment (kontrollanten) [reply orb.yml]: Yes, that was how I read it at first also. But then I saw the condition check at row 30; `<< parameters.file >>`. 

If it doesn't work, do you have any idea how to solve it? It'd been nice to have some kind of testing the verify the yaml result

Comment (kontrollanten) [reply orb.yml]: Okay. I tested it locally.

The following config:

```
jobs:
  build:
    docker:
      - image: circleci/python:3.6.1
    steps:
      - codecov/upload:
          file: coverage.xml
```

Gave the following output:

```
jobs:
  build:
    docker:
    - image: circleci/python:3.6.1
    steps:
    - run:
        name: Upload Coverage Results
        command: "bash <(curl -s https://codecov.io/bash) \\\n  -f coverage.xml \\\
          \n  -n ${CIRCLE_BUILD_NUM} \\\n  -t ${CODECOV_TOKEN} \\\n  -y .codecov.yml\
          \ \\\n  - \\\n  \n"
```

So it seems like the logic in the template works.

Comment (kontrollanten) [reply orb.yml]: I found an `-` in the beginning of line 40  that should be removed.

**Comment by thomasrockhu** [comment]
Thanks for the work @kontrollanten !

**Comment by teohhanhui** [comment]
See simpler fix in #17

**Pull request merged**